### DISCUSSION TITLE
How Do I Set Up VGMTrans?

### DISCUSSION THREAD

**Discussion by skyward406** [vgmtrans/vgmtrans#212] [Q&A] [answered]
Hello, I downloaded VGMTrans to my old laptop (back in 2019 if I remember correctly) and it worked just fine. After unzipping the folder, I was able to use the app right away. Now I have a new laptop and I can't seem to really understand what changed with the app since the last time I downloaded it. I believe it was the .bin directory I used to access the app, but it appears it is no longer there. Is there some manual setup required to use VGMTrans now? I would just like to know if there is now since there wasn't before.

**Comment by sykhro** [comment, marked as answer]
Did you download the program or the source code? The executable is in the bin folder
